src/python/gudhi/cubical_complex.pyx

```diff
@@ -145,6 +146,32 @@ cdef class CubicalComplex:
             persistence_result = self.pcohptr.get_persistence(homology_coeff_field, min_persistence)
         return persistence_result
 
+    def cofaces_of_cubical_persistence_pairs(self, homology_coeff_field=11, min_persistence=0, persistence_dim_max = False):
+        """This function returns the persistence of the simplicial complex.
```

Comment:
I think we want more description, even more than you put in the "returns" field. Some vague suggestion (split between the beginning and the returns section)

A persistence interval is described by a pair of cells, one that creates the feature and one that kills it. The filtration values of those 2 cells give coordinates for a point in a persistence diagram, or a bar in a barcode. Structurally, in the cubical complexes provided here, the filtration value of any cell is the minimum of the filtration values of the maximal cells that contain it. Connecting persistence diagram coordinates to the corresponding value in the input (i.e. the filtration values of the top-dimensional cells) is useful for differentiation purposes.
This function returns a list of intervals, where for each interval it gives a 2-tuple with its dimension and a 2-tuple of extremities, and for each extremity it gives a 2-tuple of a filtration value and the (index of the) top-dimensional cell it originates from.

Reply:
I shamelessly copy-pasted your nice doc in the file ;-) I can still it even more if you think it is still too rough